daedalus/src/Daedalus/BackendApi.purs

```diff
@@ -29,7 +30,10 @@ mkUrl :: URLPath -> URL
 mkUrl = joinWith "/"
 
 backendApi :: URLPath -> URL
-backendApi path = mkUrl $ [backendPrefix, "api"] <> path
+backendApi path = mkUrl $ [backendPrefix, "api"] <> path <> ifEmptyEnd
+  where
+    -- Workaround for passing empty passphrases as last capture in URL
+    ifEmptyEnd = if last path == Just "" then [""] else []
```

Comment:
This will be fixed by HD wallets PR where we will properly handle empty passphrase